src/main/java/org/broadinstitute/hellbender/cmdline/StandardArgumentDefinitions.java

```diff
@@ -19,6 +19,9 @@ private StandardArgumentDefinitions(){}
     public static final String VERBOSITY_NAME = "verbosity";
     public static final String READ_VALIDATION_STRINGENCY_LONG_NAME = "read-validation-stringency";
     public static final String ASSUME_SORTED_LONG_NAME = "assume-sorted";
+    public static final String READ_FILTER_LONG_NAME = "read-filter";
+    public static final String DISABLE_READ_FILTER_LONG_NAME = "disable-read-filter";
```

Comment:
ReadFilter arguments are now in https://github.com/broadinstitute/gatk/blob/master/src/main/java/org/broadinstitute/hellbender/cmdline/ReadFilterArgumentDefinitions.java